### PR TITLE
Update OSG Software accomplishments and activities

### PR DESCRIPTION
**Pull request by brianhlin**
First pass, @osg-cat

### CODE REVIEW

File: pages/projects/osg-software.md [hunk 1]
```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
     abandoned software, and develops tools to fill any gaps to provide a software stack suitable for WLCG sites.
     This software stack is distributed through packages used widely by the community, such as RPMs and container images.
 
--   **Integration testing:** the team coordinates tests of the integrated software stack among internal staff, external
-    stakeholders, and automated means.
+-   **Integration testing:** the team coordinates tests of the integrated software stack among internal staff and
+    external stakeholders.
+    Additionally, the team maintains a nightly integration test suite that covers different combinations of software and
```
Comment (osg-cat) [inline pages/projects/osg-software.md:29]: Additionally, the team maintains and monitors nightly automated tests that exercise different integrated software installations, varying software versions, installation methods, and hosting operating systems.

File: pages/projects/osg-software.md [hunk 2]
```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
...
     deployment models by distributing select production services as container images.
-    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC experiments.
-    The team follows a strategy of providing a common base container across all communities, plus community-specific refinements.
-    The team currently supports three communities, U.S. ATLAS, U.S. CMS, and the general OSG community, with [caching](caching.md) container images.
+    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC
+    experiments.
+    The team follows a strategy of providing a common base container across all communities, plus community-specific
+    refinements for U.S. ATLAS and U.S. CMS.
+
+-   **Enterprise Linux 8 support:** the OSG Software Stack is built upon the Enterprise Linux (EL) operating system
```
Comment (osg-cat) [inline pages/projects/osg-software.md:69]: Is there another useful sentence to add here about helping the community to track EL8 distributions, especially once changes to the popular CentOS 8 were announced?

File: pages/projects/osg-software.md [hunk 3]
```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
...
+    coordination of external software providers.
 
 -   **Streamlining the release process:** to get new software into the hands of site administrators more quickly, the
     team has implemented rolling-style releases along with a policy to make release candidates available more quickly.
+
+-   **Retirement of OSG 3.4:** Retirement of older software is the other side of the coin of maintaining the OSG
+    Software Stack that gives our community timelines to transition to new technologies while ensuring
+    production-readiness of the distributed computing infrastructure. In particular, the retirement of OSG 3.4 meant the
+    end of support for software such as edg-mkgridmap, GUMS, BeStMan, and VOMS Admin Server packages as well as the
+    end-of-life of RedHat Enterprise Linux 6 based operating systems.
```
Comment (osg-cat) [inline pages/projects/osg-software.md:81]: I would add, at the bottom, some variant of the usual IRIS-HEP report text about routine software releases. Treat it like an annual report and throw out some numbers for the past year (you get to define “past year”) — releases, software updates, etc. Whatever you usually include.

File: pages/projects/osg-software.md [hunk 4]
```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
...
+    experiments.
+    The team follows a strategy of providing a common base container across all communities, plus community-specific
+    refinements for U.S. ATLAS and U.S. CMS.
+
+-   **Enterprise Linux 8 support:** the OSG Software Stack is built upon the Enterprise Linux (EL) operating system
+    substrate and new operating system versions requires rebuilding and verification of the entire software stack.
+    EL 8 included a major version upgrade of Python from 2 to 3, resulting in major development efforts by the team and
+    coordination of external software providers.
 
 -   **Streamlining the release process:** to get new software into the hands of site administrators more quickly, the
```
Comment (osg-cat) [inline pages/projects/osg-software.md:74]: I wonder if this bullet is best combined with the container one. Not all DevOps is containers, right?

**Review by osg-cat — COMMENTED** [review]
I’ve made a few small comments. But I think that this is in pretty good shape, so take or leave my comments and then go ahead and merge this in. I doubt Frank will look at the GitHub source, but will look at the live webpage.

**Comment by brianhlin** [comment]
@osg-cat I agree with a bunch of these comments but won't be able to get to them until I get back from Indiana so I say we merge as is for now. I don't have the permission to merge in this repo, do you?

**Pull request merged**